test_requests.py

```diff
@@ -411,6 +416,21 @@ def test_DIGESTAUTH_QUOTES_QOP_VALUE(self):
         r = requests.get(url, auth=auth)
         assert '"auth"' in r.request.headers['Authorization']
 
+    def test_DIGESTAUTH_THREADED(self):
```

Comment:
Yeah, I've been considering creating a separate repo for a much larger set of tests. They could be run each time a commit is pushed, PR is opened, and on a periodic basis (just for the sake of paranoia), but I haven't gotten around to writing them.